other/euclidean_gcd.py

```diff
@@ -1,14 +1,39 @@
-# https://en.wikipedia.org/wiki/Euclidean_algorithm
+""" https://en.wikipedia.org/wiki/Euclidean_algorithm """
 
 
-def euclidean_gcd(a, b):
+def euclidean_gcd(a: int, b: int) -> int:
+    """
+    Examples:
+    >>> euclidean_gcd(3, 5)
+    1
+
+    >>> euclidean_gcd(6, 3)
+    3
+    """
     while b:
         t = b
         b = a % b
         a = t
```

Comment:
```suggestion
        a, b = b, a % b
```
Multiple assignment in Python eliminates the need for these temporary variables.

Reply:
Fixed